bin/setup

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env sh
+
+# Set up Rails app. Run this script immediately after cloning the codebase.
+
+# Exit if any subcommand fails
+set -e
+
+# Set up ruby version file
+if [ ! -f .ruby-version ]; then
+  cp .ruby-version.example .ruby-version
+fi
+
+# Set up configurable environment variables
+if [ ! -f .env ]; then
+  cp config/env .env
+fi
+
+# Set up Ruby dependencies via Bundler
+gem install bundler --conservative
+bundle check || bundle install --path=vendor/bundle
```

Comment:
> Vendoring is helpful when developing multiple projects

The main caveat that I'm aware of is this: newer versions of gems providing CLI executables (e.g. bundle, rspec, guard, rerun) will take precedence over the versions specified in the application's `Gemfile.lock` when run without `bundle exec`, which can cause unexpected and broken behavior.

Out of curiosity, are there other reasons you like to vendor @Insti?